Bottom: d1967ebf7ced7657fd554041ef0406285bcb2590
Top:    0a8f88f0f8bd6adf13b44b79a052225959a08578
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-24 11:50:05 +0800

Refresh of vhost_net-tx-batching

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 0138650..b9d6a45 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2601,22 +2601,6 @@ int vhost_prefetch_desc_indices(struct vhost_virtqueue *vq,
 	total = vq->avail_idx - vq->last_avail_idx;
 	ret = total = min(total, num);
 
-#if 0
-	while (total) {
-		int ret2 = vhost_get_avail(vq, indices[0],
-			   &vq->avail->ring[last_avail_idx & (vq->num - 1)]);
-		if (unlikely(ret2)) {
-			vq_err(vq, "Failed to get descriptors\n");
-			return -EFAULT;
-		}
-		heads[ret - total].id = indices[0];
-		heads[ret - total].len = 0;
-		--total;
-		++indices;
-		++last_avail_idx;
-	}
-#endif
-
 	for (i = 0; i < total; i++) {
 		int ret2 = vhost_get_avail(vq, heads[i].id,
 		    &vq->avail->ring[(last_avail_idx + i) & (vq->num - 1)]);
